src/NUnitTestAdapter/TestLogger.cs

```diff
@@ -48,7 +48,7 @@ public void LoadingAssemblyFailedWarning(string dependentAssembly, string source
 
         public void NUnitLoadError(string sourceAssembly)
         {
-            SendErrorMessage("NUnit failed to load " + sourceAssembly);
+            SendInformationalMessage("NUnit failed to load " + sourceAssembly);
```

Comment:
It seems odd that "NUnitLoadError" is not an error. Do we call it from many places? Is it sometimes an error and sometimes not? Seems like we should either rename it (if it's always used informationally) or not use it in the particular case you are trying to fix.